### PR TITLE
Section trees improve readability

### PR DESCRIPTION
**Pull request by poornimanayar**
Hi,

I have made some general improvements to readability in the documentation. Also there was a place where the routepath was wrong in context to the example. I have corrected that too.

Poornima

### CODE REVIEW

File: Extending/Section-Trees/trees.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ The first node in the tree is referred to as the **Root Node**. You might want t
 
 
 :::note
-In V8 the /config/trees.config file has been removed.
+In V8 the `/config/trees.config` file has been removed.
```
Comment (sofietoft) [inline Extending/Section-Trees/trees.md]:
```suggestion
In Umbraco 8 the `/config/trees.config` file has been removed.
```

File: Extending/Section-Trees/trees.md [hunk 2]
```diff
@@ -140,7 +140,7 @@ If you're creating a custom tree as part of an Umbraco package/plugin, it's reco
 public class FavouriteThingsTreeController : TreeController
 ```
 
-The edit view in the example would now be loaded from the location: */app_plugins/favouriteThings/backoffice/favouriteThingsAlias/edit.html*
+The edit view in the example would now be loaded from the location: */App_Plugins/favouriteThings/backoffice/favouriteThingsAlias/edit.html*
```
Comment (sofietoft) [inline Extending/Section-Trees/trees.md]:
```suggestion
The edit view in the example would now be loaded from the location: `/App_Plugins/favouriteThings/backoffice/favouriteThingsAlias/edit.html`
```

File: Extending/Section-Trees/trees.md [hunk 3]
```diff
@@ -213,7 +213,7 @@ protected override TreeNode CreateRootNode(FormDataCollection queryStrings)
     return root;
 }
 ```
-The RoutePath should be in the format of: **section/treeAlias/method**. As our example controller uses the PluginController attribute, clicking the root node would now request /App_Plugins/favouriteThings/backoffice/favouriteThingsAlias/overview.html. If you are not using the PluginController attribute, then the request would be to /umbraco/views/favouriteThingsAlias/overview.html
+The RoutePath should be in the format of: **section/treeAlias/method**. As our example controller uses the `PluginController` attribute, clicking the root node would now request **/App_Plugins/favouriteThing/backoffice/favouritistThingsAlias/overview.html**. If you are not using the `PluginController` attribute, then the request would be to **/umbraco/views/favouritistThingsAlias/overview.html**
```
Comment (sofietoft) [inline Extending/Section-Trees/trees.md]:
```suggestion
The RoutePath should be in the format of: **section/treeAlias/method**. As our example controller uses the `PluginController` attribute, clicking the root node would now request `/App_Plugins/favouriteThing/backoffice/favouritistThingsAlias/overview.html`. If you are not using the `PluginController` attribute, then the request would be to `/umbraco/views/favouritistThingsAlias/overview.html`.
```

**Comment by poornimanayar** [comment]
Haa, you picked up the `App_Plugins`. Let me know if you need help with that. I am prepping for a tak next week and I am all over documentation ATM. Can do some fixes as clearing-my-head-exercise.

**Comment by sofietoft** [comment]
That would be wonderful @poornimanayar ! 😄 
Can't really pin point exact articles with issues, but if you see some while browsing through, you're more than welcome to help out 💪 

I'll make sure to get this one merged in 😁 
Thanks again!

**Pull request merged**